configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionPreparer.java

```diff
@@ -332,6 +338,8 @@ private void writeStateToZooKeeper(SessionZooKeeperClient zooKeeperClient,
             zooKeeperClient.writeDockerImageRepository(dockerImageRepository);
             zooKeeperClient.writeAthenzDomain(athenzDomain);
             zooKeeperClient.writeQuota(quota);
+            if (dedicatedClusterControllerCluster)
+                zooKeeperClient.writeDedicatedClusterControllerCluster();
```

Comment:
You need to do the same in `SessionRepository.createSessionFromExisting()`, as per the comment on line 334